dist/tools/buildsystem_sanity_check/check.sh

```diff
@@ -87,12 +87,31 @@ check_not_exporting_variables() {
     fi
 }
 
+# Deprecated variables or patterns
+# Prevent deprecated varibles or patterns to re-appear after cleanup
+check_deprecated_vars_patterns() {
+    local patterns=()
+    local pathspec=()
+
+    patterns+=(-e 'FEATURES_MCU_GROUP')
+
+    # Pathspec with exclude should start by an inclusive pathspec in git 2.7.4
+    pathspec+=('*')
+
+    # Ignore this file when matching as it self matches
+    pathspec+=(":!${SCRIPT_PATH}")
+
+    git -C "${RIOTBASE}" grep "${patterns[@]}" -- "${pathspec[@]}" \
+        | sed '1i Deprecated variables or patterns:'
```

Comment:
I added this change in a fixup commit in https://github.com/RIOT-OS/RIOT/pull/11671

```suggestion
        | sed '1i \\nDeprecated variables or patterns:'
```

It is required to behave correctly when multiple errors are shown.

Reply:
I cherry-picked the fixup commit